content/02-getting-started/01-installing-the-cardano-node.mdx

```diff
@@ -17,6 +17,88 @@ the source code using either of the following:
 Once you have installed the node, you need to
 [specify the configuration parameters](https://github.com/input-output-hk/cardano-node/blob/master/doc/getting-started/understanding-config-files.md/).
 
+### How to Run the Cardano node and CLI using Docker
+There is also a Docker image available that you can use. 
+
+#### Setting up Docker
+
+1. Download and install Docker/Docker Desktop from the [Docker site](https://docs.docker.com/get-docker/).
+2. Ensure you have allocated enough RAM to Docker (we recommend setting the RAM to 8GB in the Docker Desktop requirements) and start the Docker daemon, as described in the Docker instructions.
+
+#### Setting up the Cardano node
+1. Download the correct `cardano-node` image:
+
+``docker image pull inputoutput/cardano-node:<TAG>``
+
+where `<TAG>` is the tag for the version that you require (e.g. `latest` for the most recent stable version, or `1.27.0` for node version 1.27.0).
+
+2. Create local `data` and `node-ipc` volumes:
+
+```
+docker image create data
+docker image create node-ipc
```

Comment:
Wrong docker command, and we should use more descriptive names for volumes so they are easily identified and reduce risk of being deleted by mistake. Replace with the following **and change throughout rest of document:**
`docker volume create cardano-node-data`
`docker volume create cardano-node-ipc`

Reply:
On second thoughts, we don't actually need these at all because volumes are created on `run` if they don't already exist. But it's still worth using more descriptive volume names.